src/dmd/cppmanglewin.d

```diff
@@ -997,7 +1000,7 @@ private:
                 saved_idents[i] = name;
                 break;
             }
-            if (!strcmp(saved_idents[i], name)) // ok, we've found same name. use index instead of name
+            if (saved_idents[i] == name) // ok, we've found same name. use index instead of name
```

Comment:
A case where the string comparison can be eliminated if it is switched to `Identifier`s instead.